build/rst2man.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+# Author:
+# Contact: grubert@users.sf.net
+# Copyright: This module has been placed in the public domain.

```

Comment:
Generally we don't put license information in every file, but if you want to keep author information, can you unify this header with rst2text.py?

Reply:
The rst2man.py is a direct copy from Docutils repo. It seems reasonable to leave it as-is.

I'll remove it from rst2txt.py, as the file is really not worth worrying about.